gem/templates/tanzania/base.html

```diff
@@ -13,7 +13,7 @@
 <!--<![endif]-->
   <head>
     <meta charset="utf-8"/>
-    <title>{% block title %}{% if self.seo_title %}{{ self.seo_title }}{% else %}{{ self.title }}{% endif %}{% endblock %}</title>
+    <title>Tujibebe | {% if self.seo_title %} {% if self.seo_title == "Main" %} Home {% else %}{{ self.seo_title }}{% endif %}{% else %} {% if self.title == "Main" %} Home {% else %}{{ self.title }}{% endif %}{% endif %}</title>
```

Comment:
@Mitso the seo page titles don't the get translated?

Reply:
if logic blocks have been changed to if blocks...

Reply:
Thanks @moh-moola , the dynamic content get the current page translation from the CMS.

Reply:
@Mitso I think this should be spread across multiple lines, it makes it easier to understand and see logical errors